packages/xsschema/src/wrap.ts

```diff
@@ -0,0 +1,29 @@
+import type { StandardSchemaV1 } from '@standard-schema/spec'
+
+export const wrapArray = async <T extends StandardSchemaV1>(schema: T): never | Promise<T> => {
+  const vendor = schema['~standard'].vendor
+  try {
+    // eslint-disable-next-line ts/no-unsafe-assignment
+    const { array } = await import(vendor)
```

Comment:
We should consider vendors that do not export `array`

Reply:
maybe throw an error to notice user that vendor do export `array`?

Reply:
> maybe throw an error to notice user that vendor do export `array`?

No, we already support arktype and will soon support Effect Schema.

They don't use a Zod-like format

https://arktype.io/docs/objects#arrays
https://effect.website/docs/schema/basic-usage/#arrays

Reply:
so I need to split it up like `to-json-schema` does?

Reply:
> yup, it's best to add unit tests as well.

will do